pkg/k8s/deployment_test.go

```diff
@@ -37,12 +42,14 @@ var _ = Describe("Operations for k8s Deployment kind", func() {
 		schema := runtime.NewScheme()
 		err := appsv1.AddToScheme(schema)
 		Expect(err).ToNot(HaveOccurred())
+		c = fake.NewFakeClientWithScheme(schema, objects...)
+		get = operator.New(c)
```

Comment:
I have a bit of a problem understanding this. You are creating new operator called `get`?

Reply:
The naming of the "operator" test helper package is a bit nonhelpful in the construct call, I agree. "getters", "helpers", not sure what to call it

Reply:
https://github.com/maistra/istio-workspace/pull/469/commits/597ac6def9ade65dd8ef08a58a84741ada556acf